nixos/modules/services/misc/turn-rs.nix

```diff
@@ -0,0 +1,58 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+
+let
+  cfg = config.services.turn-rs;
+  configFile = pkgs.writeText "config.toml" cfg.settings;
+in
+{
+  options.services.turn-rs = {
+    enable = lib.mkEnableOption "turn-rs server";
+    package = lib.mkPackageOption pkgs "turn-rs" { };
+
+    settings = lib.mkOption {
+      type = lib.types.str;
+      description = "Turn-rs server config file";
+      default = ''
```

Comment:
This should be a proper submodule type. Right now you probably need to repeat half of the default text which is not great.